src/main/java/dk/kb/present/copyright/CopyrightAccessDto2SolrFieldsMapper.java

```diff
@@ -17,27 +17,50 @@ public class CopyrightAccessDto2SolrFieldsMapper {
     private String searligeVisningsVilkaar; //Fra 'rullelisten'. (Blokeret er på rullelisten med håndteres specielt)
     private String materialeType;
     private boolean ejerMaerke;
-    private boolean vandMaerke; //PT. ikke i brug
     private boolean pligtAfleveret;
-    
+    private boolean fotoAftale;
+    private boolean billedeAftale;
+    private String ophavsretTekst; //Temporary field for jura
+    private String filNavn;//used to log which records has errors
     private String imageUrl=null; 
     public CopyrightAccessDto2SolrFieldsMapper(CopyrightAccessDto accessDto) {
```

Comment:
This is a valid point! But there is logic involved here that has changed many times, while the CopyrightAccessDto object has not changed. Also for unittest it is much easier to make a CopyrightAccessDto object for testing instead of a gigantic XML.

Reply:
Fair point about testing.